.pre-commit-config.yaml

```diff
@@ -33,3 +33,10 @@ repos:
     -   id: doc8
         args: ["doc/"]
         pass_filenames: false
+-   repo: https://github.com/koalaman/shellcheck-precommit
```

Comment:
installing trunk in this repository would come with a collection of git hooks but also increase code quality in general.